 python3 /vzwhome/udayka/workspace/accept_user_input.py \
--project vz-it-np-gudv-dev-vzntdo-0 \
--runner DataflowRunner \
--region us-east4 --staging_location gs://vznet-test/accept_user_input/stg \
--temp_location gs://vznet-test/wireline_churn_test/tmp/ \
--template_location gs://vznet-test/wireline_churn_test/code/accept_user_input
--input_bq_table test_dataset.dim_inventory_customer_profiles_norm_v0
--output_gcs_path gs://vznet-test/wireline_churn_test/tmp
usage: accept_user_input.py [-h] --project_id PROJECT_ID --input_bq_table INPUT_BQ_TABLE --output_gcs_path OUTPUT_GCS_PATH --region REGION --temp_location
                            TEMP_LOCATION --staging_location STAGING_LOCATION
accept_user_input.py: error: the following arguments are required: --input_bq_table, --output_gcs_path
--login: --input_bq_table: command not found
--login: --output_gcs_path: command not found
